hmda/src/main/resources/persistence.conf

```diff
@@ -37,6 +37,10 @@ cassandra-journal {
   ssl.truststore.password = ${?TRUSTSTORE_PASSWORD}
 }
 
+cassandra-query-journal {
+  gap-free-sequence-numbers = off
```

Comment:
Lets include the details of what this setting does 

```
  # Sequence numbers for a persistenceId is assumed to be monotonically increasing
  # without gaps. That is used for detecting missing events.
  # In early versions of the journal that might not be true and therefore
  # this can be relaxed by setting this property to off.
```

Reply:
yeap, added reference link and description